config/recipes/elastic-agent/multi-output.yaml

```
@@ -0,0 +1,235 @@
+apiVersion: agent.k8s.elastic.co/v1alpha1
+kind: Agent
+metadata:
+  name: elastic-agent
+spec:
+  version: 7.14.0
+  elasticsearchRefs:
+  - outputName: default
+    name: elasticsearch
+  - outputName: monitoring
+    name: elasticsearch-mon
+  daemonSet:
+    podTemplate:
+      spec:
+        containers:
+        - name: agent
+          volumeMounts:
+          - mountPath: /var/log
+            name: varlog
+        dnsPolicy: ClusterFirstWithHostNet
```

```suggestion
        dnsPolicy: ClusterFirstWithHostNet
        hostNetwork: true
```

otherwise we have just the Agents' container names in the data reported.